addshare/addshare.c

```diff
@@ -66,10 +87,10 @@ static const struct option opts[] = {
 	{NULL,			0,			NULL,	 0  }
 };
 
-static void show_version(void)
+static int show_version(void)
 {
 	printf("ksmbd-tools version : %s\n", KSMBD_TOOLS_VERSION);
```

Comment:
Maybe g_print instead.

Reply:
The printf statement was not changed. This is offtopic.